tests/translator/input/api_with_binary_media_types_definition_body.yaml

```diff
@@ -0,0 +1,26 @@
+Globals:
```

Comment:
Could you also add an example of this with a template containing inline swagger?

Reply:
Added